test/integration/scripting_spec.mjs

```diff
@@ -1858,8 +1857,6 @@ describe("Interaction", () => {
           expect(text).withContext(`In ${browserName}`).toEqual("00000000123");
 
           await page.click(getSelector("26R"));
-          // eslint-disable-next-line no-restricted-syntax
-          await waitForTimeout(10);
```

Comment:
@calixteman Alternatively, would replacing such occurrences with `waitForSandboxTrip` work to be more sure that any scripting actions are fully done, or is no sandbox trip performed at all in this case?